src/main/java/core/basesyntax/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String HEADER = "fruit,quantity";
+    private static final String COMMA_SEPARATOR = ",";
+    private StringBuilder builder;
```

Comment:
The `StringBuilder` instance `builder` is declared but not used in the `getReport` method. Consider removing it if it is not needed.